src/components/Pressable/GenericPressable/BaseGenericPressable.js

```diff
@@ -122,14 +122,14 @@ const GenericPressable = forwardRef((props, ref) => {
             hitSlop={shouldUseAutoHitSlop && hitSlop}
             onLayout={onLayout}
             ref={ref}
-            onPress={!isDisabled && onPressHandler}
-            onLongPress={!isDisabled && onLongPressHandler}
-            onKeyPress={!isDisabled && onKeyPressHandler}
-            onPressIn={!isDisabled && onPressIn}
-            onPressOut={!isDisabled && onPressOut}
+            onPress={!isDisabled ? onPressHandler : undefined}
+            onLongPress={!isDisabled ? onLongPressHandler : undefined}
+            onKeyPress={!isDisabled ? onKeyPressHandler : undefined}
+            onPressIn={!isDisabled ? onPressIn : undefined}
+            onPressOut={!isDisabled ? onPressOut : undefined}
             style={state => [
                 getCursorStyle(isDisabled, [props.accessibilityRole, props.role].includes('text')),
-                props.style,
+                StyleUtils.parseStyleFromFunction(props.style, state),
```

Comment:
Why this change is required?

Reply:
It is required to apply styles based on the component states like isHovered, isDisabled, etc just like the lines below it. @robertKozik told me to add it so maybe he can add more context here if needed

Reply:
The idea inside `BaseGenericPressable` was to spread styling based on Pressable state into different props (HoverStyle, focusStyle etc.). So in place of:
```javaScript
style={({isHovered, isPressed}) => ({
    isHovered && <<hoverStyle>>,
    isPressed && <<pressStyle>>
})}
```
We would get:
```javaScript
pressedStyle={<<pressStyle>>}
hoverStyle={<<hoverStyle>>}
```

But as `PressableWithFeedback` shifts all these state-aware styling into `opacityView` and passes `wrapperStyle` as the style prop to `GenericPressable`, I suggested this change in order to still have the possibility of state-aware styling inside the wrapperStyle prop.

During our convo with @priyeshshah11 I came up with this change because I thought It could be used there. 
All in all, even when there is no use here, this change is beneficial.